sites/all/modules/overdrive/overdrive.module

```diff
@@ -125,6 +125,7 @@ function overdrive_ting_covers($entities) {
 function overdrive_ding_entity_buttons($type, $entity) {
   $buttons = array();
 
+//  if (preg_match('/^publizon:/', $entity->ding_entity_id);
```

Comment:
This looks like a leftover from a debugging session.

Reply:
Removed, tried a little different things to track down the root of the issue.